RELEASE_NOTES.md

```diff
@@ -108,7 +108,7 @@ August 16, 2022
 - `browsers.webview_android.releases.108` ([#17373](https://github.com/mdn/browser-compat-data/pull/17373))
 - `browsers.webview_android.releases.109` ([#17373](https://github.com/mdn/browser-compat-data/pull/17373))
 - `browsers.webview_android.releases.110` ([#17373](https://github.com/mdn/browser-compat-data/pull/17373))
-- `css.types.image.paint.additional_paramters` ([#17314](https://github.com/mdn/browser-compat-data/pull/17314))
+- `css.types.image.paint.additional_parameters` ([#17314](https://github.com/mdn/browser-compat-data/pull/17314))
```

Comment:
This is a feature identifier, so we shouldn't modify it.

```suggestion
- `css.types.image.paint.additional_paramters` ([#17314](https://github.com/mdn/browser-compat-data/pull/17314))
```

Reply:
I believe I change that one in the JSON file too. Since it was only a few weeks old, I thought it might have been unintentional

Reply:
It was definitely an unintentional typo, however we still added it in with a typo.  When the next release comes around, its release notes will clarify that the typo version had been removed in favor of the fixed one!

Reply:
OK, but should the fix in the JSON file stay?
I'll probably ignore the RELEASE_NOTES.md file from the typo checking going forward anyway

Reply:
Yes, we should definitely change the identifier in the actual JSON file!  The reason we're not fixing the same typo in the release notes is because we want to represent that the feature was added with the broken name initially.  When this PR is merged to fix the name, the release notes will show that the identifier with the typo was removed and the fixed identifier was added!

I imagine that the release notes will still have potential for typos, but I can say with certainty that anything in the release notes in inline code (`foo`) should be ignored, if you're able to do that?